modules/fluentd/templates/fluentd.conf.erb

```diff
@@ -0,0 +1,84 @@
+<source>
+  @type exec
+  command "log stream --color none --level debug --type log --style syslog \
+      --timeout 1d \
+    | tail -n +3 \
+    | awk '{printf (/^[0-9]{4}/&&NR>1?RS:x)\"%s\",$0} END {print \"\"}'"
+    # tail to omit first 2 lines that cause syslog parsing to fail:
+    #   Filtering the log data using "type == 1024"
+    #   Timestamp                       (process)[PID]    
+    # awk remove newlines apple's log output as syslog prints in log messages
```

Comment:
To simplify this (removing the step with awk), we could output json and capture specific fields. However, Json input for fluentd expects to not have the entries in an array: some string manipulation will still be required.

Reply:
I switched to the default log style to get the log level/severity.